io-engine/src/bdev/device.rs

```diff
@@ -644,3 +684,22 @@ pub fn bdev_event_callback<T: BdevOps>(
 pub fn dispatch_loopback_removed(name: &str) {
     dispatch_bdev_event(DeviceEventType::LoopbackRemoved, name);
 }
+
+/// Call back function for the Block Device Flush
+extern "C" fn bdev_flush_complete_cb(
+    bio: *mut spdk_bdev_io,
+    result: bool,
+    arg: *mut c_void,
+) {
+    let s = unsafe { Box::from_raw(arg as *mut oneshot::Sender<bool>) };
+    if !result {
+        error!("Bdev Flush failed");
```

Comment:
do we need both logs on callback and waiter?

Reply:
removed the duplicate logs.